packages/react-native-bpk-component-navigation-bar/README.md

```diff
@@ -113,6 +113,7 @@ PropTypes.shape({
 **iOS:**
 
 + `navigationBarShadowColor`
++ `navigationBarPrimaryColor`
```

Comment:
This is now a breaking change since a theme that was previously valid is now not valid